### PR TITLE
Refactor alert admin

### PR DESCRIPTION
**Pull request by gaoyan1998**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-admin/src/main/resources/mapper/AlertRulesMapper.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    </select>
+
+
+    <select id="selectForProTable" resultType="org.dinky.data.model.AlertRule">
+        select
+        a.*
+        from
+        dinky_alert_rules a
+        <where>
+            1=1
```
Comment (leechor) [inline dinky-admin/src/main/resources/mapper/AlertRulesMapper.xml]: '1=1' isn't needed after <where> statement.

Comment (gaoyan1998) [reply dinky-admin/src/main/resources/mapper/AlertRulesMapper.xml]: > '1=1' isn't needed after statement.

The `1 = 1` is needed here, follow-up `and `operation maybe does not exist

Comment (leechor) [reply dinky-admin/src/main/resources/mapper/AlertRulesMapper.xml]: [trim, where, set](https://mybatis.org/mybatis-3/dynamic-sql.html#:~:text=MyBatis%20has%20a%20simple%20answer%20that%20will%20likely%20work%20in%2090%25%20of%20the%20cases.%20And%20in%20cases%20where%20it%20doesn%E2%80%99t%2C%20you%20can%20customize%20it%20so%20that%20it%20does.%20With%20one%20simple%20change%2C%20everything)

Comment (gaoyan1998) [reply dinky-admin/src/main/resources/mapper/AlertRulesMapper.xml]: > [trim, where, set](https://mybatis.org/mybatis-3/dynamic-sql.html#:~:text=MyBatis%20has%20a%20simple%20answer%20that%20will%20likely%20work%20in%2090%25%20of%20the%20cases.%20And%20in%20cases%20where%20it%20doesn%E2%80%99t%2C%20you%20can%20customize%20it%20so%20that%20it%20does.%20With%20one%20simple%20change%2C%20everything)

Well, it's been deleted

File: dinky-admin/src/main/java/org/dinky/controller/AlertTemplateConteroller.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    private final AlertTemplateService alertTemplateService;
+
+    @GetMapping
+    public Result<List<AlertTemplate>> list() {
+        return Result.succeed(alertTemplateService.list());
+    }
+
+    @DeleteMapping
+    public Result<Boolean> delete(int id) {
```
Comment (Zzm0809) [inline dinky-admin/src/main/java/org/dinky/controller/AlertTemplateConteroller.java:54]: added log op

File: dinky-common/src/main/resources/i18n/messages_en_US.properties [hunk 3]
```diff
@@ -136,3 +136,13 @@ file.upload.failed=File upload failed, reason: {0}
 
 hdfs.init.failed=Internal hdfs client initialization error, reason: {0}
 hdfs.file.lose=The core-site.xml/hdfs-site.xml/yarn-site.xml files were not found in the project root directory, please upload these files first
+
+
+
+# dinky-alert
+alert.rule.jobFail=Job Failure
+alert.rule.getJobInfoFail=Gte Job Info Fail
```
Comment (Zzm0809) [inline dinky-common/src/main/resources/i18n/messages_en_US.properties]: gte?  is Get ?

File: dinky-web/src/pages/RegCenter/Alert/AlertTemplate/components/AlertTemplateForm.tsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+              required: true,
+              message: l('rc.template.templateCodePlaceholder')
+            }
+          ]}
+        >
+          <CodeEdit
+            {...CodeEditProps}
+            code={codeValue}
+            onChange={(value) => setCodeValue(value ?? '')}
+            language={'markdown'}
```
Comment (Zzm0809) [inline dinky-web/src/pages/RegCenter/Alert/AlertTemplate/components/AlertTemplateForm.tsx]: 上边设置过了 language 了,重复设置了

File: dinky-web/src/pages/SettingCenter/AlertRule/AlertRuleList/RuleEditForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+  onSubmit: (values: AlertRule) => void;
+  modalVisible: boolean;
+  values: Partial<AlertRule>;
+};
+
+const RuleEditForm = (props: AlertRuleFormProps) => {
+  const {onSubmit: handleSubmit, onCancel: handleModalVisible, modalVisible, values} = props;
+
+  // if is system rule disable edit
+  const isSystem = values.ruleType == 'SYSTEM';
```
Comment (Zzm0809) [inline dinky-web/src/pages/SettingCenter/AlertRule/AlertRuleList/RuleEditForm/index.tsx]: use enum

File: script/sql/upgrade/1.0.0-SNAPSHOT_schema/mysql/dinky_ddl.sql [hunk 6]
```diff
@@ -244,6 +244,33 @@ CREATE TABLE `dinky_sys_role_menu` (
 
 
 
+drop table if exists `dinky_alert_template`;
```
Comment (Zzm0809) [inline script/sql/upgrade/1.0.0-SNAPSHOT_schema/mysql/dinky_ddl.sql]: upgrade's sql 不需要 drop 语句

Comment (Zzm0809) [reply script/sql/upgrade/1.0.0-SNAPSHOT_schema/mysql/dinky_ddl.sql]: > upgrade's sql 不需要 drop 语句

还有需要再 dml 里添加 insert 语句

**Comment by gaoyan1998** [comment]
> ## Purpose of the pull request
>    
> 
> ## Brief change log
> ## Verify this pull request
> This pull request is code cleanup without any test coverage.
> 
> _(or)_
> 
> This pull request is already covered by existing tests, such as _(please describe tests)_.
> 
> (or)
> 
> This change added tests and can be verified as follows:

@zhu-mingye PTAL

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**